chapters/projectManagement.typ

```diff
@@ -5,21 +5,17 @@
 // [x] Zeiterfassung (Stunden pro Woche/Stunden pro Task-Kategorie, wie Implementation Doku, Meeting, etc.)
 // Hinweis: Keine Liniendiagramme für die Darstellung von Zeit/Arbeitsaufwand pro Woche
 
-This section describes the approach of the project as well as the project plan and time tracking.
```

Comment:
Wür de Satz drin lah zum allgmemein z beschriebe was im kapitel vorchunt und denn im detail wo was isch.

Reply:
Hmm han eifach gfunde es tönt chli komisch, 3 mol approach uf de gliche Zile grad.

Reply:
mh jao ok müsst mer evtl. es anders wort finde

Reply:
hmm mer chonnt ned gschids in Sinn zum umschribe. Würs glaub gern weg loh.
Im gliche Absatz wird jo das alles scho gseit.